### PR TITLE
wnnlo for w and z

### PR DESCRIPTION
**Pull request by fdolek**

### CODE REVIEW

**Comment by areinsvo** [comment]
@trtomei could you review the updated trigger list included in this PR? I believe it should be in sync with the recommendations and with what mononet is doing. 

FYI @mcremone this PR also includes the NNLO weights taken from monojet.

**Comment by mcremone** [comment]
Has this been tested?

**Comment by fdolek** [comment]
it is tested just for one sample,

**Comment by mcremone** [comment]
It would be good to have a sense of the effect of this additional correction before approving the PR.

File: analysis/secondary_inputs/triggers.py [hunk 1]
```diff
@@ -1,60 +1,36 @@
...
-                             "HLT_PFMET170_NotCleaned",
-                             #"HLT_PFMET170_HBHE_BeamHaloCleaned",
-                             #"HLT_PFMETNoMu120_NoiseCleaned_PFMHTNoMu120_IDTight",
-                             #"HLT_PFMETNoMu110_NoiseCleaned_PFMHTNoMu110_IDTight",
-                             #"HLT_PFMETNoMu90_NoiseCleaned_PFMHTNoMu90_IDTight",
-                             "HLT_PFMETNoMu90_PFMHTNoMu90_IDTight",
+met_trigger_paths["2016"] = ["HLT_PFMETNoMu90_PFMHTNoMu90_IDTight",
                              "HLT_PFMETNoMu100_PFMHTNoMu100_IDTight",
                              "HLT_PFMETNoMu110_PFMHTNoMu110_IDTight",
                              "HLT_PFMETNoMu120_PFMHTNoMu120_IDTight"]
```
Comment (trtomei) [inline analysis/secondary_inputs/triggers.py]: "HLT_PFMETNoMu120_PFMHTNoMu120_IDTight" is the only one that was unprescaled the whole year. We may add the others, but I would prefer to have only the 120 one such that we have a "simple" turn on curve.

File: analysis/secondary_inputs/triggers.py [hunk 2]
```diff
@@ -1,60 +1,36 @@
...
 met_trigger_paths["2017"] = ["HLT_PFMETNoMu120_PFMHTNoMu120_IDTight_PFHT60",
```
Comment (trtomei) [inline analysis/secondary_inputs/triggers.py]: Notice that even adding "HLT_PFMETNoMu120_PFMHTNoMu120_IDTight_PFHT60" to the mix doesn't get us the full luminosity. There are still 6/pb missing from runs 298997, 300673 and 305766. Those runs were a mess: some paths were prescaled by mistake, some paths had the wrong L1 seed mix... :(

File: analysis/secondary_inputs/triggers.py [hunk 3]
```diff
@@ -1,60 +1,36 @@
...
-                             "HLT_PFMETNoMu120_PFMHTNoMu120_IDTight",
-                             "HLT_PFMETNoMu130_PFMHTNoMu130_IDTight",
-                             "HLT_PFMETNoMu140_PFMHTNoMu140_IDTight"]
+                             "HLT_PFMETNoMu120_PFMHTNoMu120_IDTight"]
 met_trigger_paths["2018"] = ["HLT_PFMETNoMu120_PFMHTNoMu120_IDTight_PFHT60",
-                             "HLT_PFMETNoMu120_PFMHTNoMu120_IDTight",
-                             "HLT_PFMETNoMu140_PFMHTNoMu140_IDTight"]
+                             "HLT_PFMETNoMu120_PFMHTNoMu120_IDTight"]
```
Comment (trtomei) [inline analysis/secondary_inputs/triggers.py]: Same here. MET120 and MET120_HT60 should essentially be what we need.

**Pull request closed without merging**